api/types/types.go

```diff
@@ -155,3 +155,27 @@ type Info struct {
 	Name               string
 	Labels             []string
 }
+
+type ContainerState struct {
```

Comment:
I want us to decide which way to go or else we'll forget about it and start building cruft again :(